network-api/networkapi/wagtailpages/models.py

```diff
@@ -242,6 +242,24 @@ class Petition(CTA):
         blank=True
     )
 
+    share_twitter = models.CharField(
```

Comment:
might also want to add `editable=False` to `share_link` and `share_link_text` (and also remove it from the admin) so that we aren't allowed to use that field anymore but at the same time maintaining backwards-compatibility for existing data (+ add a note as to why we're doing this)